coalib/output/ConsoleInteraction.py

```diff
@@ -607,6 +609,138 @@ def print_actions(console_printer, section, actions, failed_actions):
     return get_action_info(section, actions[choice - 1], failed_actions)
 
 
+def try_to_apply_action(action_name,
+                        chosen_action,
+                        console_printer,
+                        section,
+                        metadata_list,
+                        action_dict,
+                        failed_actions,
+                        result,
+                        file_diff_dict,
+                        file_dict):
+    """
+    Try to apply the given action.
+
+    :param action_name:     The name of the action.
+    :choose_action:         The action object that will be applied.
+    :param console_printer: Object to print messages on the console.
+    :param section:         Currently active section.
+    :param metadata_list:   Contains metadata for all the actions.
+    :param action_dict:     Contains the action names as keys and their
+                            references as values.
+    :param failed_actions:  A set of all actions that have failed. A failed
+                            action remains in the list until it is successfully
+                            executed.
+    :param result:          Result corresponding to the actions.
+    :param file_diff_dict:  If it is an action which applies a patch, this
```

Comment:
it's a dictionary with differences for a file. This is the description copied from other function already implemented.